backend/src/controllers/tasks.controller.ts

```diff
@@ -28,6 +29,32 @@ export class TasksController {
     return this.tasksService.insertTask(task);
   }
 
+  @Post('/new-comment/:taskId')
+  async insertComment(
+    @Param('taskId') taskId: number,
+    @Body() comment: Comment,
+  ): Promise<void> {
+    if (!comment.comment) {
+      console.log('Comment is missing.');
+      return;
+    }
+    if (comment.comment.length > 255) {
+      console.log('Comment is too long.');
+      return;
```

Comment:
In order to follow the REST specification and standardize the communication between the frontend and the backend, can we return the HTTP status code 400 Bad Request if the request body does not pass the validation criteria?